skypy/power_spectrum/_growth.py

```diff
@@ -159,21 +159,18 @@ def integrand(x):
         integrand = (growth_factor(x, cosmology, gamma) - 1) / (1 + x)
         return integrand
 
-    if isinstance(z, int) or isinstance(z, float):
-        integral = integrate.quad(integrand, z, 1100)[0]
-        g = np.exp(integral)
-        growth_function = g / (1 + z)
-
-    elif isinstance(z, np.ndarray):
-        growth_function = np.zeros(np.shape(z))
+    z_flat = np.ravel(z)
+    g_flat = np.empty(z_flat.size)
 
-        for i, aux in enumerate(z):
-            integral = integrate.quad(integrand, aux, 1100)[0]
-            g = np.exp(integral)
-            growth_function[i] = g / (1 + aux)
+    for i, zz in enumerate(z_flat):
```

Comment:
```suggestion
    for i, z in enumerate(z_flat):
```
could now just rename this variable `z`?